NEWS.md

```diff
@@ -13,8 +13,19 @@ low-resolution version of such files is automatically downloaded when a
 higher-resolution version is not available. Please, refer to
 [ClimaArtifacts](https://github.com/CliMA/ClimaArtifacts) for more information.
 
+### A higher resolution land-sea mask is now used and automatically downloaded - PR [#1006](https://github.com/CliMA/ClimaCoupler.jl/pull/1006)
+
+A 60 arcsecond land-sea mask constructed from topographic data is now used.
+Topographic data is automatically downloaded and a land-sea mask is constructed
+by identifying where elevation is greater than 0. Note, this can lead to
+misidentification of ocean in some areas of the globe that are inland but below
+sea level (Dead Sea, Death Valley, ...).
```

Comment:
Do we have a plan to correct this ocean misidentification in the future?

Reply:
@juliasloan25 One of two options is feasible here 
(1) Write / use existing tools for closed contour detection to address such areas within continent boundaries, 
(2) Use a merged dataset with the ETOPO2022 information based on the existing NCAR dataset (lower resolution 360x180 npts), which already contains classifications as follows: 
`0=ocean, 1=land, 2=lake, 3=small island, 4=ice shelf`, see here: https://www.ncl.ucar.edu/Document/Functions/Shea_util/landsea_mask.shtml

Reply:
Okay! Could you open an issue for it so we can keep track of resolving this in the future?

Reply:
@juliasloan25 Please see #1049